script/build/linux-entrypoint

```diff
@@ -2,14 +2,35 @@
 
 set -ex
 
-TARGET=dist/docker-compose-$(uname -s)-$(uname -m)
-VENV=/code/.tox/py37
+CODE_PATH=/code
+VENV=${CODE_PATH}/.tox/py37
 
-mkdir -p `pwd`/dist
-chmod 777 `pwd`/dist
+cd ${CODE_PATH}
+mkdir -p dist
+chmod 777 dist
 
-$VENV/bin/pip install -q -r requirements-build.txt
+${VENV}/bin/pip3 install -q -r requirements-build.txt
+
+# TODO(ulyssessouza) To check if really needed
 ./script/build/write-git-sha
-su -c "$VENV/bin/pyinstaller docker-compose.spec" user
-mv dist/docker-compose $TARGET
-$TARGET version
+
+export PATH="${CODE_PATH}/pyinstaller:${PATH}"
```

Comment:
See previous comment regarding no need for {} here

Reply:
I think it's generally good practice to be consistent and use `{}` (but there might be two camps on that)

Reply:
Yes, there is (two camps). :-)
When I see a variable like that and there is no obvious thing in the variable name that calls for having the {}, I think ah, there must be some substitution or such going on and then where it isn't I see that oh, that was just somebody using {} when it was not needed. In simple use lite $PATH, one immediately recognizes that there is a regular PATH and moves on directly.

Reply:
https://github.com/docker/compose/pull/6641#discussion_r274918137